sources/lib/util/options.ml

```diff
@@ -848,8 +857,9 @@ let case_split_policy () = !M.case_split_policy
 let instantiate_after_backjump () = !M.instantiate_after_backjump
 let disable_weaks () = !M.disable_weaks
 let minimal_bj () = !M.minimal_bj
+let cdcl_tableaux_inst () = !M.cdcl_tableaux_inst
+let cdcl_tableaux_th () = !M.cdcl_tableaux_th
 let tableaux_cdcl () = !M.tableaux_cdcl
```

Comment:
export an auxiliary function `cdcl_tableaux () = !M.cdcl_tableaux_inst || !M.cdcl_tableaux_th`, and use it in `satml.ml` and `satml_frontend.ml` when relevant